tests/pc/test_po_update.py

```diff
@@ -67,54 +81,54 @@ def test_po_update(duthosts, enum_rand_one_per_hwsku_frontend_hostname, tbinfo):
 
         # Step 1: Remove portchannel members from portchannel
         for member in portchannel_members:
-            duthost.shell("config portchannel member del %s %s" % (portchannel, member))
+            duthost.shell("config portchannel %s member del %s %s" % (namespace_prefix, portchannel, member))
```

Comment:
can you reuse https://github.com/Azure/sonic-mgmt/blob/a632af5d00bd82edd1f831b872bfddad5b4cac29/tests/common/devices/sonic_asic.py#L460
```
asichost.config_portchannel_member(...)
```
Make changes there if required.